albumentations/augmentations/transforms.py

```diff
@@ -3107,3 +3108,46 @@ def apply(self, img, transforms=(), **params):
 
     def get_transform_init_args_names(self):
         return ("brightness", "contrast", "saturation", "hue")
+
+
+class Emboss(ImageOnlyTransform):
+    """Emboss the input image and overlays the result with the original image.
+
+    Args:
+        alpha ((float, float)): range to choose the visibility of the embossed image. At 0, only the original image is
+            visible,at 1.0 only its embossed version is visible. Default: (0.2, 0.5).
+        strength ((float, float)): strength range of the embossing. Default: (0.2, 0.7).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+    """
+
+    def __init__(self, alpha=(0.2, 0.5), strength=(0.2, 0.7), always_apply=False, p=0.5):
+        super(Emboss, self).__init__(always_apply, p)
+        self.alpha = to_tuple(alpha, 0.0)
+        self.strength = to_tuple(strength, 0.0)
```

Comment:
Invalid usage of `to_tuple`. Valid values for `alpha` in range `[0, 1]`, but if we will provide single value, for example `alpha=0.5` we will get `alpha=(-0.5, 0.5)`.
Correct usage: `to_tuple(alpha, low=0)`

Reply:
The same thing for `strength`. `strength` must be in range `[0, inf)`

Reply:
I am wrong, it is correct usage because `low` the second argument.